app/services/submit_reference.rb

```diff
@@ -38,10 +38,14 @@ def progress_applications
       reference_feedback_provided!
       application_form.application_choices.awaiting_references.each do |application_choice|
         ApplicationStateChange.new(application_choice).references_complete!
-        if application_form.apply_again?
-          application_choice.update(edit_by: Time.zone.now)
+        if application_form.apply_again? && !FeatureFlag.active?('move_edit_by_to_application_form')
+          application_choice.update!(edit_by: Time.zone.now)
```

Comment:
```suggestion
        if application_form.apply_again?
          application_choice.update!(edit_by: Time.zone.now)
          application_form.update!(edit_by: Time.zone.now)
```

Reply:
Yea i was trying to make it clear what should be stripped out later on, but it's just made it way more confusing to parse. Thanks for all the feedback!